bindings/python/.evergreen/test.sh

```diff
@@ -22,6 +24,10 @@ if [ "Windows_NT" = "$OS" ]; then # Magic variable in cygwin
              "C:/python/Python38/python.exe"
              "C:/python/Python39/python.exe"
              "C:/python/Python310/python.exe")
+
+    export CSFLE_PATH=../csfle/bin/mongo_csfle_v1.dll
+    C:/python/Python310/python.exe drivers-evergreen-tools/.evergreen/mongodl.py --component csfle \
+    --version 6.0.0-rc4 --out $(cygpath -m ../csfle/)
```

Comment:
Let's replace 6.0.0-rc4 with "latest" here and below. That way we don't need to update this code on each release.

Reply:
Done.